cypress/integration/search/search_group_post.js

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/*eslint max-nested-callbacks: ["error", 3]*/
+
+import users from '../../fixtures/users.json';
+
+/**
+ * create new group DM channel
+ * @param {String} {String} text - DM channel name1, DM channel name2
+ */
+
+function createNewPrivateGroupChannel(channelname1, channelname2) {
```

Comment:
Since this test is about search, can you leverage the API (https://api.mattermost.com/#tag/channels%2Fpaths%2F~1channels%2Fpost) to create the private group channel?